### ISSUE TITLE
[Feature Request]: 

### ISSUE THREAD

**Issue by DozNele** [jasp-stats/jasp-issues#2245]
### Description

Export complete R syntax of a session (or even former sessions combined)

### Purpose

Documentation, continuous analyses (for student)

### Is your feature request related to a problem?

No

### Is your feature request related to a JASP module?

Unrelated

### Describe the solution you would like

 Option to write out Rsyntax of all calculations done during one session (not copy and paste as I have to do it now). Syntax could then be loaded into JASP at the beginning of a new session and students (I am a teacher) could continue with/adapt their analyses. 

### Describe alternatives that you have considered

copy-paste all single Rcommands into a writing program and re-paste them into different analyses windows again.

**Comment by boutinb** [comment]
Yes this is definitely a feature we will add, probably this year.
